psi4/src/psi4/lib3index/df_helper.cc

```diff
@@ -1341,183 +1575,194 @@ void DF_Helper::transform_disk() {
 #ifdef _OPENMP
         rank = omp_get_thread_num();
 #endif
-        std::vector<double> Cp(nao * wMO);
+        std::vector<double> Cp(nao * wtmp);
         C_buffers[rank] = Cp;
         eri[rank] = std::shared_ptr<TwoBodyAOInt>(rifactory->eri());
     }
 
-    // scope these declarations
-    {
-        // declare bufs
-        std::vector<double> M;
+// scope buffer declarations
+{
+    // declare bufs
+    std::vector<double> T; T.reserve(max_block*nao*wtmp);
+    std::vector<double> F; F.reserve(max_block*wfinal);
+    std::vector<double> N; N.reserve(max_block*wfinal);
+    double *Tp = &T[0];
+    double *Fp = &F[0];
+    double *Np = &N[0];
```

Comment:
Yes, I have been eager to revert back to get proper alignment.  I often use `data()` for pointer assignment, but I was not consistent with this when I first started using `std::vector` for this code.